Makefile

```diff
@@ -52,6 +52,22 @@ libs: ./include/rule_type.h src/parser.c src/dialect.c $(SRC_FILES) src/Makefile
 	cd src; $(MAKE) CC=$(CC) $@
 .PHONY: libs
 
+libs_so: ./include/rule_type.h src/parser.c src/dialect.c $(SRC_FILES) src/Makefile
+	cd src; $(MAKE) CC=$(CC) $@
+.PHONY: libs_so
+
+install: libs_so
```

Comment:
I'm not sure that the install and uninstall should be part of this makefile and make assumptions on where to install. If so the install and include directory should be defined in variables, so the they can be edited in one place (or overridden from the command line).

Reply:
If there are install and uninstall rules, then shouldn't they also install the "bin/gherkin" CLI-binary?

Reply:
Ok.  This was requested by @aslakhellesoy because of things I said in the README file for Cucumber.ml project.  I am not wedded to it because I have not used make in years and am (as you can see) fairly rusty at it.

Reply:
We can remove install and uninstall, but I thought having them would simplify consumption for other developers and users.

Reply:
Hardcoding things like "/usr/lib/" on several lines deep in makefile rules is not a good idea. The `src/Makefile` is deliberately written with make variable at the top, so that is should be straight forward for a user to modify those make variables for the special c compiler that the user might use. Any install/uninstall rules should also be appropriately parameterized ("/usr/lib/" does not work on Windows for instance).